src/components/table/header/style.css

```
@@ -1,13 +1,9 @@
 :host {
 	display: table-cell;
+	padding: 0.5em;
 	padding-top: 0.75rem;
 	padding-bottom: 0.75rem;
```

- As a general rule it's better to use `rem` then `em` 
  - `rem` is relative to the font size of the `<html>` (root element) of the website. 
  - `em` is relative to the font-size of its direct or closest parent. 
  - https://www.w3schools.com/cssref/css_units.asp
- To set top-bottom and left-right padding can be done in one line instead of three. No need to clutter the css.
  - https://developer.mozilla.org/en-US/docs/Web/CSS/padding

```suggestion
	padding: 0.75rem 0.5rem;
```